src/test/java/alfredtest/TaskListTest.java

```diff
@@ -0,0 +1,36 @@
+package alfredtest;
+
+import alfred.action.TaskList;
+import alfred.task.Task;
+import alfred.task.ToDo;
+import alfred.task.Deadline;
+import alfred.task.Event;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+
+    @Test
+    public void testAddTaskToTaskList() {
```

Comment:
```suggestion
    public void addTaskToTaskListTest() {
```

Reply:
Would putting the Test at the end of the method be clearer for the user?

Reply:
Thanks! Will make the necessary changes.